modules/alma/includes/alma.reservation.inc

```diff
@@ -247,7 +247,7 @@ function alma_reservation_create($account, $id, $options = array(), $expiry = NU
     $int_per = variable_get('ding_reservation_disable_interest_date', '');
 
     // Interest period.
-    if ($int_per) {
```

Comment:
How is this change related to the issue at hand?

I see that we retrieve a value called ding_reservation_*disable*_interest_date which would infer that if the variable has a nonempty value then interest dates should be disabled. The current implementation infers that is is the other way around.

This change reverses this but this would in practice also mean that anyone who has intentionally set `ding_reservation_disable_interest_date` to enable interest dates would have interest perioids enabled.

If this is to be changed then I would like a separate issue for this.